ros2/src/geoscenario_server/geoscenario_server/geoscenario_server.py

```diff
@@ -36,7 +39,7 @@ def __init__(self):
         GSServerBase.__init__(self)
 
         parameters = [
-            Parameter(name='scenario_files', type=ParameterType.PARAMETER_STRING_ARRAY, description='GeoScenario file paths', default_value=['']),
+            Parameter(name='scenario_files', type=ParameterType.PARAMETER_STRING, description='GeoScenario file paths (comma-separated)', default_value=''),
```

Comment:
This breaks the test_ros2_server.bash script:
```
rclpy.exceptions.InvalidParameterTypeException: Trying to set parameter 'scenario_files' to '['scenarios/test_scenarios/gs_all_vehicles_peds.osm']' of type 'STRING_ARRAY', expecting type 'STRING'
```
I'd rather not change this parameter type. What's wrong with using the string array?